scripts/mirrorImages.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Declare source image project id
+declare projectid=$1
+shift
+
+# Declare and populate array of source and destination image names
+declare -A imagearray=()
+for arg; do
+  eval `echo "imagearray$arg"`
```

Comment:
Avoid using `eval` in bash scripts

This should be eliminated by just defining the function as constants intead of passing as args

Reply:
Use the whole image as array key or value